modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java

```diff
@@ -101,6 +126,21 @@ public ObjectEntry deleteObjectEntry(long objectEntryId)
 		return objectEntryLocalService.deleteObjectEntry(objectEntryId);
 	}
 
+	@Override
+	public ObjectEntry deleteObjectEntry(
```

Comment:
Usually we get the objectEntry by external reference code and then call delete. It is better like this but I will also call it deleteObjectEntryByExternalReferenceCode to follow the same convention as the get... methods

Reply:
Changed in the rebase.